src/components/FiltersOverlay/Filter/Toggle/index.tsx

```diff
@@ -0,0 +1,59 @@
+import styled from "@emotion/styled";
+import Button, { Variant } from "@leafygreen-ui/button";
+import { size } from "constants/tokens";
+
+interface ToggleProps {
+  value: boolean;
+  onChange: (newValue: boolean) => void;
+  ["data-cy"]?: string;
+  leftText: string;
+  rightText: string;
+}
+
+const Toggle: React.FC<ToggleProps> = ({
```

Comment:
Have you explored? Not suggesting it just asking what you think?
https://www.mongodb.design/component/segmented-control/documentation/